cylc/flow/network/scan.py

```diff
@@ -165,7 +165,7 @@ def scan_many(items, methods=None, timeout=None, ordered=False):
         try:
             msg = await client.async_request(method)
         except ClientTimeout as exc:
-            LOG.exception("ClientTimeout")
+            LOG.exception("ClientTimeout - reg: {reg}, host: {host}, port: {port}".format(reg=reg, host=host, port=port))
```

Comment:
The line width is set to 80 in pycodestyle. Bot sure if the build will pass. Might be easier with f-strings